src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -219,38 +219,25 @@ internal void SetExperimentalFeatures(ConfigScope scope, string featureName, boo
 
         internal bool IsImplicitWinCompatEnabled()
         {
-            bool? settingValue = ReadValueFromFile<bool?>(ConfigScope.CurrentUser, DisableImplicitWinCompatKey);
-            if (!settingValue.HasValue)
-            {
-                // if the setting is not mentioned in configuration files, then the default DisableImplicitWinCompat value is False
-                settingValue = ReadValueFromFile<bool?>(ConfigScope.AllUsers, DisableImplicitWinCompatKey, defaultValue: false);
-            }
+            bool settingValue = ReadValueFromFile<bool?>(ConfigScope.CurrentUser, DisableImplicitWinCompatKey)
+                ?? ReadValueFromFile<bool?>(ConfigScope.AllUsers, DisableImplicitWinCompatKey)
+                ?? false;
 
-            return !settingValue.Value;
+            return !settingValue;
         }
 
         internal string[] GetWindowsPowerShellCompatibilityModuleDenyList()
         {
-            string[] settingValue = ReadValueFromFile<string[]>(ConfigScope.CurrentUser, WindowsPowerShellCompatibilityModuleDenyListKey);
-            if (settingValue == null)
-            {
-                // if the setting is not mentioned in configuration files, then the default WindowsPowerShellCompatibilityModuleDenyList value is null
-                settingValue = ReadValueFromFile<string[]>(ConfigScope.AllUsers, WindowsPowerShellCompatibilityModuleDenyListKey);
-            }
-
-            return settingValue;
+            return ReadValueFromFile<string[]>(ConfigScope.CurrentUser, WindowsPowerShellCompatibilityModuleDenyListKey)
+                ?? ReadValueFromFile<string[]>(ConfigScope.AllUsers, WindowsPowerShellCompatibilityModuleDenyListKey)
+                ?? null;
```

Comment:
Why not:
```suggestion
                ?? ReadValueFromFile<string[]>(ConfigScope.AllUsers, WindowsPowerShellCompatibilityModuleDenyListKey);
```

Reply:
I wanted to state the default value explicitly, so the code could be self-documenting.

Reply:
I think it makes no sense in the case.